fsh/ig-data/input/pages/3_howtoJoin.md

```diff
@@ -39,6 +39,19 @@ To fill the Store of the Bridgehead you need access to clinical data from your *
 The Store of the Bridgehead stores data as **HL7 FHIR® Bundles**. To import data into the Store, they must be harmonised and converted. Each site must build an ETL-process (extract, transform, load) to harmonise the structure of the data and translate them to fit predefined FHIR® profiles.
 You can find our FHIR® profile for the basic data set [here](https://simplifier.net/bbmri.de) and for the oncology data set [here](https://simplifier.net/oncology).
 
+6. Definition of "Sample"
+A sample, as undestood by the Sample Locator, should satisfy the following conditions:
+
+* Samples for which all parameters are identical are considered to be a single sample and will counted only once - or- may only be loaded once into the store. & 
+* Samples that differ in at least one parameter (central or not) are considered to be different samples and are both counted - or uploaded to the store.
```

Comment:
Each sample can have multiple aliquots, should they treat each aliquot as an individual sample or not?

Reply:
I would write:
>Aliquots for which all must-support parameters from the profile are identical are considered to be a single sample and should be loaded only one in the store.

We can't match parameter-equal samples during a query and count all only as one. That would be too costly. So every sample loaded in the bridgehead is counted if it matches, even if there are multiple parameter-equal instances.